dotCMS/src/main/java/com/dotcms/variant/VariantFactoryImpl.java

```diff
@@ -80,37 +87,61 @@ public void update(Variant variant) throws DotDataException {
                 .addParam(variant.archived())
                 .addParam(variant.identifier())
                 .loadResult();
+
+        CacheLocator.getVariantCache().put(variant);
```

Comment:
We need to invalidate when a variant is updated, not put.  We only send invalidations around in a cluster.  Loading from cache always needs to be done lazily.